mappings/net/minecraft/world/World.mapping

```diff
@@ -1,16 +1,57 @@
 CLASS net/minecraft/class_18 net/minecraft/world/World
+	FIELD field_179 lightingQueueCount I
 	FIELD field_180 isRemote Z
+	FIELD field_181 lightingQueue Ljava/util/List;
+	FIELD field_182 entitiesToUnload Ljava/util/List;
+	FIELD field_183 scheduledUpdates Ljava/util/TreeSet;
+	FIELD field_184 scheduledUpdateSet Ljava/util/Set;
+	FIELD field_185 blockEntityUpdateQueue Ljava/util/List;
+	FIELD field_186 worldTimeMask J
+	FIELD field_187 timeLoaded J
+	FIELD field_188 allPlayersSleeping Z
+	FIELD field_189 tempCollisionBoxes Ljava/util/ArrayList;
+	FIELD field_190 processingDeferred Z
+	FIELD field_191 lightingUpdateCount I
+	FIELD field_192 allowMonsterSpawning Z
+	FIELD field_193 allowMobSpawning Z
+	FIELD field_194 activeChunks Ljava/util/Set;
+	FIELD field_195 ambientSoundCounter I
+	FIELD field_196 tempEntityList Ljava/util/List;
+	FIELD field_197 instantTick Z
```

Comment:
The MCP name for this is `scheduledUpdatesAreImmediate` should we change the name of this to reflect that it is a boolean rather than an integer tick count. Would it be better named `instantTicksEnabled` or something?

Reply:
`instantBlockUpdateEnabled` probably